### DISCUSSION TITLE
Unable to redirect to specified uri

### DISCUSSION THREAD

**Discussion by Sagarmahalle** [coder/code-server#2214] [Q&A]
Hi All, 

I have one website https://<domain name>/ui?t=yeftdrrk5f#mrogkvo625

and I want to redirect the request to above uri. 

I am not able to pass "/ui?t=yeftdrrk5f#mrogkvo625" in proxy pass.

**Comment by code-asher** [comment]
I don't think NGINX has any problems with passing along query variables
via proxy_pass. You shouldn't have to do anything special as far as I'm
aware.

**Reply by Sagarmahalle** [reply]
I tried..but didn't work.
Looks like special character issue

**Reply by code-asher** [reply]
Interesting. You might have better luck posting on Stack Overflow but if you want to post your config I'll see if I can replicate the issue.
